src/components/controls/TextArea/TextArea.tsx

```diff
@@ -57,16 +64,23 @@ export const TextArea = React.forwardRef<HTMLSpanElement, TextAreaProps>(functio
         onUpdate,
         onChange,
     } = props;
+
+    const {errorMessageProp, validationStateProp} = errorPropsMapper({
```

Comment:
It make sense to invert names here, function will accept all props and will return `errorMessage` and `validationState` without *Prop